pkgs/top-level/all-packages.nix

```diff
@@ -7796,6 +7800,10 @@ with pkgs;
 
   belle-sip = callPackage ../development/libraries/belle-sip { };
 
+  libbfd = callPackage ../development/libraries/libbfd { };
+
+  binutilsOpcodes = callPackage ../development/libraries/binutils-opcodes { };
```

Comment:
`opcodes`? `libopcodes`? `binutils-opcodes`? Help me bikeshed!

Reply:
Yes, `binutils-opcodes`. We don't do camelCase in package names anymore.

Reply:
> We don't do camelCase in package names anymore.

Oh, glad I asked! I saw a vague trend but had no idea that was the policy. Kebob FTW!